specification/cognitiveservices/data-plane/LUIS/Authoring/stable/v2.0/LUIS-Authoring.json

```diff
@@ -2815,11 +2815,23 @@
         ],
         "responses": {
           "201": {
-            "description": "An object containing the application endpoint URL and its assigned endpoint key.",
+            "description": "An object containing the application endpoint URL, its assigned endpoint key and publishing status.",
             "schema": {
               "$ref": "#/definitions/ProductionOrStagingEndpointInfo"
             }
           },
+          "207": {
+            "description": "An object containing the application endpoint URL, its assigned endpoint key and publishing status.",
+            "schema": {
+              "$ref": "#/definitions/ProductionOrStagingEndpointInfo"
+            }
+          },
+          "503": {
```

Comment:
We don't document 5xx responses for now

Reply:
But this one is thrown on purpose, it is not an arbitrary exception. Should it be removed too?